ts/features/bonus/bpd/screens/details/components/summary/__test__/bpdSummaryComponent.test.tsx

```diff
@@ -34,6 +34,12 @@ import {
 } from "../../../../../store/reducers/__mock__/ranking";
 import BpdSummaryComponent from "../BpdSummaryComponent";
 
+jest.mock("react-native-safe-area-context", () => ({
+  useSafeAreaInsets: jest
+    .fn()
+    .mockReturnValue({ top: 20, left: 0, right: 0, bottom: 0 })
+}));
+
```

Comment:
What's the modification in this PR that needs this mock? Maybe the bottomsheet?

Reply:
Correct. Since the bottomsheet has now a dependency on this hook, we have to mock it in tests